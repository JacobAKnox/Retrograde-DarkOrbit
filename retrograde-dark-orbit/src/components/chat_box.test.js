import { render } from '@testing-library/react'
import ChatBox from './chat_box';
import "setimmediate";

it('chat_box component test', () => {
    const page = render(<ChatBox/>);
    expect(page).toMatchSnapshot();
});