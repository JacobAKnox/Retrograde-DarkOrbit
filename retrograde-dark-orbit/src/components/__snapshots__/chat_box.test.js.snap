// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`chat_box component test 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="border-4 border-blue-400 min-w-[600px] min-h-[600px] bg-gray-800 m-20 p-4 rounded-xl flex-col"
      >
        <div
          class="bg-gray-700 w-full h-[600px] p-2 rounded-xl overflow-y-auto"
          id="message-container"
        />
        <div
          class="bg-inherit items-center justify-start w-full h-10 mt-4 flex justify-between"
        >
          <input
            aria-label="message"
            class="bg-slate-700 text-white w-[415px] h-10 rounded-xl p-2"
            placeholder="message"
            value=""
          />
          <button
            class="bg-slate-700 text-white w-32 h-10 rounded-xl hover:bg-slate-600 disabled:bg-slate-950 disabled:text-gray-700"
          >
            Send
          </button>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="border-4 border-blue-400 min-w-[600px] min-h-[600px] bg-gray-800 m-20 p-4 rounded-xl flex-col"
    >
      <div
        class="bg-gray-700 w-full h-[600px] p-2 rounded-xl overflow-y-auto"
        id="message-container"
      />
      <div
        class="bg-inherit items-center justify-start w-full h-10 mt-4 flex justify-between"
      >
        <input
          aria-label="message"
          class="bg-slate-700 text-white w-[415px] h-10 rounded-xl p-2"
          placeholder="message"
          value=""
        />
        <button
          class="bg-slate-700 text-white w-32 h-10 rounded-xl hover:bg-slate-600 disabled:bg-slate-950 disabled:text-gray-700"
        >
          Send
        </button>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
